docs/developers-clear-containers-install.md

```diff
@@ -5,7 +5,23 @@ instructions for any developers eager to try Clear Containers 3.0 and who
 want to build Clear Containers from the source code and are familiar with the
 process.
 
-## Requirements
+## Initial setup
+
+The recommended way to create a development environment is to first install the
+packaged versions of the Clear Containers components to create a working
+system:
+
+  * [Fedora](fedora-installation-guide.md)
+  * [Ubuntu](ubuntu-installation-guide.md)
+
+The instructions will install all packaged components, plus docker,
+the hypervisor and the Clear Containers image and kernel.
+
+After validating it is possible to create a Clear Container (the final
+step in the installation guides), progress to installing the desired
+component(s) you wish to develop.
```

Comment:
This sentence is hard to follow. "Progress to" can be interpreted as 'follow instructions up to', or 'next, install' - I suggest we break it into two parts following the closing parenthesis and replace "progress to".

Reply:
On reflection, I think we can just drop that entire paragraph.